schema/pet-insurance.json

```diff
@@ -461,45 +1216,45 @@
                         },
                         "firstName": {
                             "type": "string",
-                            "description": "",
-                            "example": "Jon"
+                            "description": "First name(s) or given name(s), as specified in ID.",
+                            "minLength": 1,
+                            "example": "Bob"
                         },
                         "middleName": {
                             "type": "string",
                             "default": "",
-                            "description": ""
+                            "description": "Middle name, if applicable.<br/>This will only be used on websites which provide separate entry for middle names, otherwise it will be ignored.<br/>If middle name is essential for placing order, consider appending it to <code>firstName</code>."
                         },
                         "lastName": {
                             "type": "string",
-                            "description": "",
-                            "example": "Doe"
+                            "description": "Last name or surname, as specified in ID.",
+                            "minLength": 1,
+                            "example": "Smith"
                         },
                         "dateOfBirth": {
                             "type": "string",
-                            "description": "Date of birth in YYYY-MM-DD format.",
+                            "description": "Proposer's date of birth in YYYY-MM-DD format.",

```

Comment:
Is this "owner" or "proposer"? These have different legal meaning as far as insurance goes.